### PR TITLE
CLAMAV_NEEDED env var cleanup

### PR DESCRIPTION
**Pull request by abottoms-coder**
## Summary of Changes
_Provide a brief summary of changes_
When incorporating these changes into the deployed environments w/o the env var, this would fail as a required env var. We've made the env var more permissive and updated the deployment scripts which should automatically set that moving forward.

## How to Test

**Ensure your `tdrs-backend/.env` does NOT have CLAMAV_NEEDED.**
```
export TDRS_HOME=`pwd`
source commands.sh
tdrs-start
```
0. Ensure the application doesn't puke w/o having CLAMAV_NEEDED as a env var.

```
echo "CLAMAV_NEEDED=True" >> tdrs-backend/.env
tdrs-start-av
```

2. Open http://localhost:3000/ and sign in.
1. Go to data files and upload a test file
3. Check in Django Admin console that the file was uploaded and that the clamav scan happened successfully.
4. Shutdown all containers w/ `tdrs-stop` (you'll have to kill the clamav one manually, sorry :( )
5. Change .env to set CLAMAV_NEEDED to `False`
6. Start all containers and ensure standard functionality
```
tdrs-start
```
## Tabletop

- [x] Ready for Tabletop (Reviewer check this)
- [x] Tabletop scheduled (Assignee check this)
- [x] Tabletop approval 

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x]  Ensure containers are started with clamav or without depending on alias
+ [x]  Ensure app functions with either setup as expected
+ [x]  Try both combinations without setting CLAMAV_NEEDED variable

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?

  + [x] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [x] Are frontend code style checks passing on CircleCI?
+ [x] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [x] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

_this deploys fine and can confirm that the env var is set to True but there are issues authenticating in qasp deployed site. beyond scope of this ticket. 

### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [x] Does this PR provide background for why coding decisions were made?
+ [ ] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [ ] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [x] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented?

### CODE REVIEW

**Comment by abottoms-coder** [comment]
Merged in code from @riatzukiza 's PR for 1692 for testing that files are still scanned. Can spin up with the new alias `tdrs-start-av` or the old alias `tdrs-start` w/o clamav. Neither run owasp/zap.

**Review by jorgegonzalez — APPROVED** [review]

File: tdrs-backend/tdpservice/settings/common.py [hunk 1]
```diff
@@ -294,7 +294,7 @@ class Common(Configuration):
     ###
 
     # Flag for local testing to enable AV Scans
-    CLAMAV_NEEDED = get_required_env_var_setting('CLAMAV_NEEDED')
+    CLAMAV_NEEDED = os.getenv('CLAMAV_NEEDED', True)
```
Comment (riatzukiza) [inline tdrs-backend/tdpservice/settings/common.py:297]: This wasn't working in the false case. It seems like it is seeing a string of length greater than 0 and accepting that as true. I found this solution:
```suggestion
    CLAMAV_NEEDED = bool(os.getenv('CLAMAV_NEEDED', 'True').lower() in ('true', '1', 't'))
```

Source: https://stackoverflow.com/questions/63116419/evaluate-boolean-environment-variable-in-python

**Review by riatzukiza — CHANGES_REQUESTED** [review]

**Comment by abottoms-coder** [comment]
@riatzukiza Are there still changes you want or is this ready for tabletop?

**Review by riatzukiza — APPROVED** [review]
LGTM

**Review by ADPennington — APPROVED** [review]
👍🏾 apologies the delay. had to troubleshoot spinning up the frontend locally.

**Pull request merged**